saskatoon/sitebase/templates/app/base/header.html

```diff
@@ -1,99 +1,125 @@
-                                    <i class="fa fa-caret-down"></i>
+                    {% else %}
+                        <li>
+                            <a href='{% url 'login' %}'>
+                                <span class="menu-it-icon-pro">
+                                    <span class="h5">{% translate 'login' %}</span>
+                                    <i class="notika-icon notika-support"></i>
                                 </span>
-                                <form action="{% url 'set_language' %}" method="post"
-                                      style="display: none" id="change_language_form">
-                                    {% csrf_token %}
-                                    <input type="hidden" value="" name="language" id="language" />
-                                </form>
                             </a>
-                            <ul class="dropdown-menu">
-                                {% for language in languages %}
-                                <li><a lang_code="{{ language.code }}" class="change_language">
-                                    {{ language.name_local|title }} ({{ language.code }})
-                                </a></li>
-                                {% endfor %}
-                            </ul>
                         </li>
-                    </ul>
-                </div>
+                    {% endif %}
+                    <li class="nav-item">
+                        <a href="#"
+                           data-toggle="dropdown"
+                           role="button"
+                           aria-expanded="false"
+                           class="nav-link dropdown-toggle">
+                            <span class="menu-it-icon-pro h5">
+                                <span class="language-title">{% translate "Languague" %}:</span>
+                                {{ LANGUAGE_CODE }}
+                                <i class="fa fa-caret-down"></i>
+                            </span>
+                            <form action="{% url 'set_language' %}"
+                                  method="post"
+                                  style="display: none"
+                                  id="change_language_form">
+                                {% csrf_token %}
+                                <input type="hidden" value="" name="language" id="language" />
+                            </form>
+                        </a>
+                        <ul class="dropdown-menu">
+                            {% for language in languages %}
+                                <li>
+                                    <a lang_code="{{ language.code }}" class="change_language">{{ language.name_local|title }} ({{ language.code }})</a>
+                                </li>
+                            {% endfor %}
+                        </ul>
+                    </li>
+                </ul>
             </div>
         </div>
     </div>
 </div>
+</div>
```

Comment:
is this extra </div> tag intentional?

Reply:
good catch. the extra tag was actually an `<a>` and my formatter skipped the indent for it